src/components/Table/TableComponents/TableFilter.tsx

```diff
@@ -15,15 +17,21 @@ type TableFilterProps<T extends Record<string, any>> = {
 	instance: TableInstance<T>;
 };
 
+type FilterColumns = {
+	id: string;
+	hdr: string | undefined | null;
+	checked: boolean | "mixed";
+};
+
 // biome-ignore lint/suspicious/noExplicitAny: we require maximum flexibility here
 export const TableFilter = <T extends Record<string, any>>({
 	translations,
 	instance,
 }: TableFilterProps<T>) => {
 	// translations
-	const clear = translations.clear || defaultTranslations.toolbar.clear;
-	const close =
-		translations.close || defaultTranslations.toolbar.close || "Close";
+	const apply = translations.apply || defaultTranslations.toolbar.apply;
+	const cancel =
+		translations.cancel || defaultTranslations.toolbar.cancel || "Cancel";
```

Comment:
Remove secondary `||` as it is redundant.
```suggestion
		translations.cancel || defaultTranslations.toolbar.cancel;
```

Reply:
I know that it's not your code (not really) but we must have missed this in previous PRs, and you're touching the code, so we should improve as we go. The whole [Tidy First](https://learning.oreilly.com/library/view/tidy-first/9781098151232/) ideology.

Reply:
agreed. done.